### PR TITLE
Updating to a new prompt

### PR DESCRIPTION
**Pull request by baruchoxman**

### CODE REVIEW

File: src/streamlit_app.py [hunk 1]
```diff
@@ -472,7 +612,7 @@ def append_content(parent, content, arr):
     if arr != None:
         arr.append(content)
 
-    if content["debug"] and not DEBUG:
```
Comment (krakov) [inline src/streamlit_app.py:475]: why remove the DEBUG  ? if in debug, want to show the below

File: src/streamlit_app.py [hunk 2]
```diff
@@ -447,19 +582,24 @@ def process_response(response):
 
 # @st.fragment()
 def show_query_result(parent, df, resp_val, sql):
-    data_tab, explain_tab, hd_sql_tab = parent.tabs(["Data", "Explain", "SQL"])
+    data_tab, chart_tab, explain_tab, hd_sql_tab = parent.tabs(
+        ["Data", "Chart", "Explain", "SQL"]
```
Comment (krakov) [inline src/streamlit_app.py:586]: i think that the chart and data are better together - or at least, the chart to come first imho

File: src/streamlit_app.py [hunk 3]
```diff
@@ -419,26 +555,25 @@ def process_response(response):
...
+            if "limit " not in order_fmt.lower():
+                sql += f"\nLIMIT {RESULTS_LIMIT}"
 
         except Exception as e:
             # If bad semantic query (for example LLM hallucinated a field) - ignore and continue
             st.error(f"Can't get SQL: {e}\n{native_app_sql}")
             continue
 
-        if DEBUG:
-            st.code(sql)
```
Comment (krakov) [inline src/streamlit_app.py:440]: do we have other way of debugging this?

File: src/streamlit_app.py [hunk 4]
```diff
@@ -36,97 +38,229 @@
...
+  - only metrics can be referenced in the metrics array
+  - present the rewritten question highlighting the attributes, metrics & filters you've successfully matched to the user query
+  - present the JSON query.
+
+### Response Structure
+
+```json
+{{
+  "group_by": ["entity.attribute_name"],
+  "metrics": ["entity.metric_name"],
```
Comment (krakov) [inline src/streamlit_app.py:84]: metrics are optional for data retriveal questions

File: src/streamlit_app.py [hunk 5]
```diff
@@ -36,97 +38,229 @@
...
+
+### Successful Queries
+
+#### Example 1: Total Revenue
+
+**User:** "What's the total revenue for Q1 2024 per country?"
+
+**Response:**
+
+**Working on** *What is the total of `sales.revenue` by `operation.country` where `date.quarter` is `Q1 2024`?*
```
Comment (krakov) [inline src/streamlit_app.py:146]: i like the working on - perhaps we can show it?

**Review by krakov — COMMENTED** [review]
see in comments

File: src/streamlit_app.py (hunk 4 above)
Comment (baruchoxman) [reply src/streamlit_app.py]: Fixed

File: src/streamlit_app.py (hunk 5 above)
Comment (baruchoxman) [reply src/streamlit_app.py:150]: Fixed

File: src/streamlit_app.py (hunk 2 above)
Comment (baruchoxman) [reply src/streamlit_app.py]: Fixed

**Review by krakov — APPROVED** [review]

**Pull request merged**